org.civicrm.reqangular/src/common/services/crm-ang.service.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-env amd */
+
+(function (CRM) {
+  define([
+    'common/lodash',
+    'common/modules/services'
+  ], function (_, services) {
+    'use strict';
+
+    services.factory('crmAngService', function () {
```

Comment:
do we need the `Service` suffix?

Reply:
We did that in https://github.com/civicrm/civihr/blob/staging/org.civicrm.reqangular/src/common/services/notification.service.js for example